### PR TITLE
Add empty message capability

### PR DESCRIPTION
**Pull request by KasunPerera1995**
Add the ability to publish a message with an empty payload when header frame is received.

### CODE REVIEW

File: modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/codec/InMemoryMessageAggregator.java [hunk 1]
```diff
@@ -138,4 +148,25 @@ public Message popMessage() {
     public void setTransaction(BrokerTransaction transaction) {
         this.transaction = transaction;
     }
+
+    public void publishMessage(ChannelHandlerContext ctx, AmqpChannel channel, int channelID) {
+        Message publishMessage = popMessage();
+        // flow manager should always be executed through the event loop
+        ctx.fireChannelRead((BlockingTask) () -> {
```
Comment (Asitha) [inline modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/codec/InMemoryMessageAggregator.java]: ctx.fireChannel code segment should go into header frame IMO

Comment (KasunPerera1995) [reply modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/codec/InMemoryMessageAggregator.java]: I will correct it ayye

Comment (KasunPerera1995) [reply modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/codec/InMemoryMessageAggregator.java]: There seems to be a test failure calling both popMessage() inside ctx.fireChannel code. Is it better to add popMessage() and then publish() inside ctx.fireChannel code in header frame and content frame?

**Comment by anupama-pathirage** [comment]
This repository is no longer maintained.

Issue reports and pull requests will not be attended.

**Pull request closed without merging**